### PR TITLE
RS 477 - Add Glean iOS Focus and Klar to search metrics

### PR DESCRIPTION
**Pull request by alekhyamoz**
As requested in https://mozilla-hub.atlassian.net/browse/RS-477.

### CODE REVIEW

File: sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/query.sql [hunk 1]
```diff
@@ -452,6 +452,60 @@ metrics_org_mozilla_klar AS (
...
+    metrics.labeled_counter.browser_search_ad_clicks AS search_ad_clicks,
+    metrics.labeled_counter.browser_search_in_content AS search_in_content,
+    metrics.labeled_counter.browser_search_with_ads AS search_with_ads,
+    client_info.first_run_date,
+    ping_info.end_time,
+    ping_info.experiments,
+    metrics.counter.browser_total_uri_count,
+    client_info.locale,
+  FROM
+    org_mozilla_ios_klar.metrics AS org_mozilla_klar_metrics
```
Comment (scholtzan) [inline sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/query.sql:507]: you might need to specify the project here
```suggestion 
    `moz-fx-data-shared-prod.org_mozilla_ios_klar.metrics AS org_mozilla_klar_metrics`
```

**Review by scholtzan — APPROVED** [review]

File: tests/sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/org_mozilla_ios_focus.metrics.schema.json [hunk 2]
```diff
@@ -0,0 +1,191 @@
+[
+    {
+      "fields": [
+        {
+          "type": "STRING",
+          "name": "android_sdk_version"
```
Comment (ANich) [inline tests/sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/org_mozilla_ios_focus.metrics.schema.json:6]: Probably leftover from copy-paste

File: tests/sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/org_mozilla_ios_klar.metrics.schema.json [hunk 3]
```diff
@@ -0,0 +1,190 @@
+[
+  {
+    "fields": [
+      {
+        "type": "STRING",
+        "name": "android_sdk_version"
```
Comment (ANich) [inline tests/sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/org_mozilla_ios_klar.metrics.schema.json:6]: Leftover from copy-paste

**Review by ANich — APPROVED** [review]
LGTM. I think there's a few mismatches in the schema to resolve though

File: tests/sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/org_mozilla_ios_focus.metrics.schema.json (hunk 2 above)
Comment (alekhyamoz) [reply tests/sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/org_mozilla_ios_focus.metrics.schema.json:6]: This column is present in the ios focus/klar tables. I assumed this has been done for standardization purposes.

Comment (Dexterp37) [reply tests/sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/org_mozilla_ios_focus.metrics.schema.json:6]: Yes, I can confirm that sadly that's the case :)

Comment (ANich) [reply tests/sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/org_mozilla_ios_focus.metrics.schema.json:6]: Gotcha 😅

**Pull request merged**